frontend/lib/src/components/widgets/DataFrame/hooks/useTableSizer.ts

```diff
@@ -46,6 +51,23 @@ export type AutoSizerReturn = {
   setResizableSize: React.Dispatch<React.SetStateAction<ResizableSize>>
 }
 
+/**
+ * Helper function to get the configured height from the heightConfig and element.
+ * This handles both the new heightConfig and legacy height fields.
+ */
+function getConfiguredHeight(
```

Comment:
nit: maybe also move this to `arrowUtils` for consistency (since it also contains getConfiguredWidth`